docs/components/arm/lite6.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Configure an xArmLite Arm"
```

Comment:
remember the alias

Reply:
Alias is already present, right? @npentrel am I looking at the right thing?

Reply:
Oh true. Alright then